### DISCUSSION TITLE
Einschaltdauer funktioniert nicht

### DISCUSSION THREAD

**Discussion by tromanski** [OpenCCU/OpenCCU#1362] [🇩🇪 Fragen&Antworten]
Hallo Experten,

beim Übertragen meiner CCU2 auf Raspberrymatic (3.59.6.20210703) habe ich festgestellt, dass die Funktion der "Einschaltdauer" nicht mehr funktioniert. Das jeweilige Gerät schaltet nach der angegebenen Zeit einfach nicht ab, was es unter der CCU2 gemacht hat. Habt ihr eine Idee woran das liegen könnte?

Viele Grüße

**Comment by Baxxy13** [comment]
Deine Problembeschreibung ist leider zu unpräzise!
Was ist "das jeweilige Gerät"? - HM / HmIP?
Wo wird die Einschaltdauer gesetzt? WebUI-Programm / DV / Script?

**Reply by tromanski** [reply]
Hallo Baxxy13,
vielen Dank für deine Rückmeldung. Du hast natürlich recht, dass ich nicht sehr präzise war.

Ich verwende einen HM-MOD-Re-8 zur Steuerung meiner Bewässerungsventile. Zur Reichweitenerhöhung benutze ich für den Garten eine ausgediente CCU2 als Gateway (CCU2GW0001 HomeMatic RF-LAN Gateway).
Es handelt sich um ein WebUI-Programm, indem ich die Einschaltdauer als Variable (Systemzustand) übergebe (siehe Bild).
![image](https://user-images.githubusercontent.com/59796315/128547094-1b370dd0-5d1d-46b0-9306-7304004b3038.png)

Ich hoffe, dass diese Informationen weiterhelfen.

**Reply by Baxxy13** [reply]
Ok, HM-Geräte und vor allem die Gateways sind jetzt nicht so meins.
Die "Faustregel" besagt aber... Erst die Einschaltdauer setzen und dann verzögert Einschalten. 

Also zum testen mal die Verzögerung in der zweiten Zeile auf 6s stellen.
Hilft das nicht, dann zum testen mal die Einschaltdauer direkt (und nicht über die SV) setzen. (um die SV als Fehlerquelle auszuschließen)

**Reply by tromanski** [reply]
Leider funktionieren beide Ansätze nicht
1) Einschaltdauer setzen und dann verzögert Einschalten
2) Einschaltdauer direkt setzen
Auch das deaktivieren des Gateways hat nichts zum gewünschten Erfolg geführt...

**Reply by Baxxy13** [reply]
Schade. 
Was du noch versuchen kannst:

- Programm löschen und komplett neu anlegen (keine Kopie benutzen)
- zum Testen eine DV mit virtueller Taste (und konfigurierter Einschaltdauer) erstellen
- versuchen ob's per Script klappt (Script testen, Name des Kanals korrekt eintragen)

```
string aktor_kanalname = "HM-MOD-Re-8 Kanal:1";
channels.Get (aktor_kanalname).DPByHssDP ("ON_TIME").State(30.0);
channels.Get (aktor_kanalname).DPByHssDP ("STATE").State(1);
```

Funktioniert das alles nicht gibt's noch die eQ-3 konforme "Problembehandlung":
das Teil aus der Zentrale löschen (mit Werksreset) und neu anlernen...

**Reply by tromanski** [reply]
Leider führen alle Ansätze und auch der Werksreset mit neu anlernen nicht zum Erfolg. Auch bei einem HmIP-BDT funktioniert die Einschaltdauer nicht. 
Kann ich irgendwie prüfen, ob die Einschaltdauer überhaupt an das Gerät übertragen wird?

**Reply by Baxxy13** [reply]
Wenn nicht einmal die eQ-3 Methode hilft fällt mir leider nichts mehr ein.

Klassische HM-Aktoren zeigen bei aktiver Einschaltdauer ein Zahnrad-Symbol in der WebUI beim Kanal.
Klassische HM-Aktoren mit Status-LED (bspw. HM-ES-PMSw1-Pl), da blinkt die LED wenn eine Einschaltdauer aktiv ist.
Für IP-Geräte ist mir keine Möglichkeit bekannt eine Einschaltdauer zu detektieren.

**Comment by tromanski** [comment]
Ich habe den Fehler gefunden... 
Es liegt an meiner Konfiguration in der Redmatic. Sobald ich Redmatic ausschalte funktioniert die Einschaltdauert.
Das Problem ist eine Schleife (siehe Bild) die es mir ermöglichte das Relais entweder über das Dashboard oder die Homematic zu schalten. Wenn ich die Schleife entferne, funktioniert es.

Gibt es eine Möglichkeit das Relais über das Dashboard oder Homematic zu schalten und die Einschaltdauer zu behalten?

![image](https://user-images.githubusercontent.com/59796315/129488744-5051ce44-850e-44c9-9e06-85c904b609b7.png)

**Reply by Baxxy13** [reply]
Ich denke zu RedMatic wirst du hier keine Antworten bekommen. 
Am besten du machst einen Thread dazu im [Homematic-Forum / RedMatic](https://homematic-forum.de/forum/viewforum.php?f=77) auf.
